test/TensorFlow/no_copy.swift

```diff
@@ -184,14 +184,14 @@ struct Classifier {
   }
 }
 
-public func mnist() {
+public func mnist() -> Float {
   // Training data
   let images = Tensor<Float>(randomNormal: [10, 784])
   let labels = Tensor<Float>(randomNormal: [10, 10])
   var classifier = Classifier()
   let loss = classifier.train(images: images, labels: labels,
                               learningRate: 0.3, epochCount: 100)
-  print(loss)
+  return loss
```

Comment:
@mhong Is this change Okay? Because `loss` is `Float` and cannot be passed to `_hostOp`.

Reply:
ah right. we only have a version for tensors: https://github.com/apple/swift/blob/2b0fbd0be430cbcd89652e431c521a967f8daea3/stdlib/public/TensorFlow/Utilities.swift#L283

do we want to add a similar printScalar method, and mark it @inline(never)? One option is to put it in test utils in https://github.com/apple/swift/blob/2b0fbd0be430cbcd89652e431c521a967f8daea3/stdlib/private/TensorFlowUnittest/TensorFlowUnittest.swift. 

If there are more unexpected issues, please feel free to defer/skip such work. Thanks for the help!

Reply:
Well, that makes sense. I'd like to do it in a separate PR later, since it is not directly relevant to this topic and may need to touch a number of test files.

Reply:
Sure thing. Thanks!

Reply:
I changed it back to `print` for this PR. Will clean them up later in a separate patch.